deflate.h

```diff
@@ -276,6 +276,7 @@ typedef struct internal_state {
  * IN assertion: there is enough room in pending_buf.
  */
 #define put_byte(s, c) {s->pending_buf[s->pending++] = (c);}
+#define put_block(s, b, l) {memcpy(&s->pending_buf[s->pending], (b), (l)); s->pending += (l);}
```

Comment:
Looks good, but I it is easy to misread `l` vs. `1`, also I would use multi-line macro.